src/server_manager/install_scripts/do_install_server.sh

```diff
@@ -117,6 +117,8 @@ function cloud::add_encoded_kv_tag() {
   cloud::add_tag "kv:${key}:${value}"
 }
 
+echo "true" | cloud::add_encoded_kv_tag "outline"
```

Comment:
We already have a "shadowbox" label to indicate Outline servers. I think having two labels will be confusing.

If you want to indicate install state, we should be more explicit about it, since the name is not expressing what it's representing.

Reply:
OK, changed to `"install-started"`.